components/alerts/alerts.css

```diff
@@ -0,0 +1,62 @@
+:root{
+    --primary-bg-color: #B3E5FC;
+    --secondary-bg-color: #E0E0E0;
+    --danger-bg-color:#FFCDD2;
+    --success-bg-color: #C8E6C9;
+    --warning-bg-color:#FFE0B2;
+}
+
+.alert {
+    width: 100%;
+    max-width: max-content;
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    justify-content: space-between;
+    flex-wrap: wrap;
+    gap: 1rem;
+
+    word-wrap: break-word;
+    
+    padding: 0.5rem;
+    margin: 1rem;
+    font-weight: bolder;
```

Comment:
Instead of mentioning the font-weight value as a word you can pass a value inside this attribute and then use it making it easy to understand

Reply:
passed the numeric value for font-weight